lib/floe/runner/container_runner.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative "container_runner/docker_mixin"
+require_relative "container_runner/docker"
+require_relative "container_runner/kubernetes"
+require_relative "container_runner/podman"
+
+module Floe
+  class Runner
+    class ContainerRunner
+      class << self
+        def cli_options(optimist)
```

Comment:
the original code had the user interface (cli) separate from the actual code
Does it make sense to keep these separate?

Reply:
also, can you add a dummy bash/echo runner or something?
Think this will let us know if this `cli_options` introduction actually does simplify things.

Reply:
Maybe?  plugins should own their own code and that includes the cli options in my opinion.  Whether it's in floe/runner/container_runner.rb vs floe/runner/container_runner/cli.rb, I could go either way.  For this PR, I'd keep it as is, and that could be a future cleanup, IMO.